src/Spectre.Angular2Client/src/app/app.module.ts

```diff
@@ -26,11 +26,18 @@ import { routing } from './app.routing';
 
 import { AppComponent } from './app.component';
 import { PreparationListComponent } from './preparation-list/preparation-list.component';
-import { PreparationService } from './preparation-list/preparation.service';
+import { PreparationService } from './preparation/preparation.service';
```

Comment:
Here's the one thing I was not sure about: are we moving services into external directory, containing all of them (and therefore keeping them reusable), or are we keeping them in module / component directories? How is it usually done? Are they supposed to be one-time disposable services, with single application over the project?